model/load_balancer.rb

```diff
@@ -6,15 +6,18 @@ class LoadBalancer < Sequel::Model
   many_to_one :project
   many_to_many :vms
   many_to_many :active_vms, class: :Vm, left_key: :load_balancer_id, right_key: :vm_id, join_table: :load_balancers_vms, conditions: {state: "up"}
-  many_to_many :vms_to_dns, class: :Vm, left_key: :load_balancer_id, right_key: :vm_id, join_table: :load_balancers_vms, conditions: Sequel.~(state: Sequel.any_type(["evacuating", "detaching"], :lb_node_state))
+  many_to_many :vms_to_dns, class: :Vm, left_key: :load_balancer_id, right_key: :vm_id, join_table: :load_balancer_vm_port, conditions: Sequel.~(state: Sequel.any_type(["evacuating", "detaching"], :lb_node_state)), select: Sequel.lit("DISTINCT vm.*")
```

Comment:
You can use `distinct: true` instead . `Sequel.lit` is a nice backdoor when Sequel cannot do what you need directly, but it's best to only use it in such circumstances.